src/components/reportRepair/address.js

```diff
@@ -21,10 +21,12 @@ const Address = ({handleChange, nextStep, values}) => {
           <FormGroup>
             <InputField name="address" input={input} >Address</InputField>
           </FormGroup>
+          <Link as={RouterLink} to="not-eligible">Can not find my address</Link>
```

Comment:
The link text doesn't match the [prototype](https://southwark-repairs-alpha-master.herokuapp.com/beta/v2/address) - it should be "I can't find my address on this list"
```suggestion
          <Link as={RouterLink} to="not-eligible">I can't find my address on this list</Link>
```